stonesoup/metricgenerator/uncertaintymetric.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+from .base import MetricGenerator
+from ..types.state import State, StateMutableSequence
+from ..types.metric import SingleTimeMetric, TimeRangeMetric
+from ..types.time import TimeRange
+
+
+class UncertaintyMetric(MetricGenerator):
+    """
+    Computes the covariance matrix of the state for each track as a measure of the
```

Comment:
Be clear the metric is computed at a single instance for a collection of states from different tracks. It's the sum of the matrix norms of each state at that instance. The metric generator will return this number for each instance (e.g. each timestep) in the track(s).

Reply:
Would this be better - "Computes the sum of the covariance matrix norms of each state at a time step. The metric generator will return this value at each time step in the track(s) as a measure of the uncertainty." ?